docs/content/preview/reference/configuration/insecure-yugabyted-examples.md

```diff
@@ -0,0 +1,939 @@
+```
+
+Start the second and the third node on two separate VMs as follows:
+
+```sh
+./bin/yugabyted start --advertise_address=<host-ip> --join=<ip-address-first-yugabyted-node> --cloud_location=aws.us-east.us-east-2a --fault_tolerance=zone
+```
+```sh
+./bin/yugabyted start --advertise_address=<host-ip> --join=<ip-address-first-yugabyted-node> --cloud_location=aws.us-east.us-east-3a --fault_tolerance=zone
+```
+
+After starting the yugabyted processes on all the nodes, configure the data placement constraint of the cluster:
+
+```sh
+./bin/yugabyted configure data_placement --fault_tolerance=zone
+```
+
+This command automatically determines the data placement constraint based on the `--cloud_location` of each node in the cluster. If there are 3 or more zones available in the cluster, the configure command configures the cluster to survive at least 1 availability zone failure. Otherwise, it outputs a warning message.
+
+The replication factor of the cluster defaults to 3.
+
+You can set the data placement constraint manually using the `--constraint_value` flag, which takes the comma-separated value of `cloud.region.zone`. For example:
+
+```sh
+./bin/yugabyted configure data_placement --fault_tolerance=zone --constraint_value=aws.us-east.us-east-1a,aws.us-east.us-east-2a,aws.us-east.us-east-3a
+```
+
+You can set the replication factor of the cluster manually using the `--rf` flag. For example:
+
+```sh
+./bin/yugabyted configure data_placement --fault_tolerance=zone --constraint_value=aws.us-east.us-east-1a,aws.us-east.us-east-2a,aws.us-east.us-east-3a --rf=3
+```
+
+### Create a multi-region cluster
+
+To create a multi-region cluster, start the first yugabyted node by running the `yugabyted start` command, pass in the `--cloud_location` and `--fault_tolerance` flags to set the node location details as follows:
+
+```sh
+./bin/yugabyted start --advertise_address=<host-ip> --cloud_location=aws.us-east.us-east-1a --fault_tolerance=region
+```
+
+Start the second and the third nodes on two separate VMs as follows:
+
+```sh
+./bin/yugabyted start --advertise_address=<host-ip> --join=<ip-address-first-yugabyted-node> --cloud_location=aws.us-west.us-west-1a --fault_tolerance=region
+```
+```sh
+./bin/yugabyted start --advertise_address=<host-ip> --join=<ip-address-first-yugabyted-node> --cloud_location=aws.us-central.us-central-1a --fault_tolerance=region
+```
+
+After starting the yugabyted processes on all the nodes, configure the data placement constraint of the YugabyteDB cluster as follows:
+
+```sh
+./bin/yugabyted configure data_placement --fault_tolerance=region
+```
+
+This command determines the data placement constraint based on the `--cloud_location` of each node in the cluster. If there are 3 or more regions available in the cluster, the command configures the cluster to survive at least 1 region failure. Otherwise, it outputs a warning message.
```

Comment:
The preceding command determines the data placement constraint based on the `--cloud_location` of each node in the cluster. If there are three or more regions available in the cluster, the command configures the cluster to survive at least one region failure. Otherwise, it outputs a warning message.